resolvers/locationPin.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+const db = require("../models");
+
+const locationPinResolvers = {
+  Query: {
```

Comment:
can we add another query that fetches all pins for a given eventid (something like `pinsForEvent(eventId)`) - in general we will only care about pins for a specific event and care less about getting all pins for all events